uint8 msgType
bool spinDirection
int16 maxSpeed
int32 angleControl